koch.nim

```diff
@@ -476,9 +475,9 @@ proc hostInfo(): string =
   "hostOS: $1, hostCPU: $2, int: $3, float: $4, cpuEndian: $5, cwd: $6" %
     [hostOS, hostCPU, $int.sizeof, $float.sizeof, $cpuEndian, getCurrentDir()]
 
-proc runCI(cmd: string) =
+proc runCIRaw(cmd: string) =
   doAssert cmd.len == 0, cmd # avoid silently ignoring
-  echo "runCI:", cmd
+  echo "runCIRaw:", cmd
```

Comment:
How hard can it be to put spaces after colons. It's quite common in written English.